src/query/query.ts

```diff
@@ -137,14 +175,14 @@ export class Query {
    */
   public load(params: QueryParams = {}): void {
     Object.keys(params).forEach(key => {
-      if (key === 'nostats') {
-        this.noStats = !!params.nostats;
+      if (key === 'stats') {
+        this.stats = !!params.stats;
```

Comment:
This is just a question because I don't have the full context of this part: is this part necessary?

Reply:
Removed